bin/sail

```diff
@@ -526,6 +527,18 @@ elif [ "$1" == "root-shell" ] || [ "$1" == "root-bash" ]; then
         sail_is_not_running
     fi
 
+# Initiate a MongoDB Shell within the "mongodb" container...
+elif [ "$1" == "mongodb" ]; then
+    shift 1
+
+    if [ "$EXEC" == "yes" ]; then
+        ARGS+=(exec)
+        [ ! -t 0 ] && ARGS+=(-T)
+        ARGS+=(mongodb mongosh --username=${MONGODB_USERNAME} --password=${MONGODB_PASSWORD} --authenticationDatabase=admin)
```

Comment:
Env vars read from the `.env` file of the project.
But I could use `MONGODB_URI`. If people want to set credentials, they will have to set all the variables `MONGODB_URI`, `MONGODB_USERNAME` and `MONGODB_PASSWORD`.

Reply:
Updated to use `MONGODB_URI`.